src/main/java/spoon/support/compiler/jdt/PositionBuilder.java

```diff
@@ -224,8 +224,8 @@ SourcePosition buildPositionCtElement(CtElement e, ASTNode node) {
 
 			// Handle lambda parameters without explicit type
 			if (variableDeclaration instanceof Argument && variableDeclaration.type == null) {
-				declarationSourceStart = findPrevNonWhitespace(contents, 0, declarationSourceStart - 1);
-				declarationSourceEnd = findNextNonWhitespace(contents, contents.length - 1, declarationSourceEnd + 1);
```

Comment:
@pvojtechovsky was there a reason for `declarationSourceStart - 1` and `declarationSourceEnd + 1` that I am missing?